gflownet/envs/base.py

```diff
@@ -1066,6 +1066,7 @@ def top_k_metrics_and_plots(
             name = "train ground truth"
             reward = energy_reward
         elif energy is None and reward is None:
+            # TODO: fix this
```

Comment:
Is it worth documenting what is broken here? At glance, I can't really tell.

Reply:
I don't know, to be honest. I haven't looked deeper into it, which is why I haven't fixed it. I will just fix it soon once I will actually need it for the crystals experiments.